nl_nsgi/nl_nsgi_README.txt

```diff
@@ -5,24 +5,50 @@ The files in this section originate from [NSGI](https://www.nsgi.nl/) with corre
 
 ## Included grids
 
-### Netherlands: RD and NAP -> ETRS89
+### European Netherlands: RD and NAP -> ETRS89
 
-*Source*: [NSGI](https://www.nsgi.nl/rdnaptrans)  
+*Source*: [NSGI](https://www.nsgi.nl/coordinatenstelsels-en-transformaties/coordinatentransformaties/rdnap-etrs89-rdnaptrans)  
 *Format*: GeoTIFF converted from NTv2 & GTX  
 *License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)  
 
-Variant 1:
+#### Variant 1:
 
 * nl_nsgi_rdcorr2018.tif
 * nl_nsgi_nlgeo2018.tif
 
-Recommended variant, applies the datum transformation as a separate step using
-a 3D similarity transformation.
+Recommended variant, applies the datum transformation as a separate step using a 3D similarity transformation.
 
-Variant 2:
+#### Variant 2:
+
+N.A. only relevant for PROJ 4
+
+#### Hybrid variant:
 
 * nl_nsgi_rdtrans2018.tif
+* nl_nsgi_nlgeo2018.tif
 
 Includes the datum transformation in the correction grid.
 
 Refer to the [RDNAPTRANS™2018 documentation](https://www.nsgi.nl/rdnaptrans) for details.
+
+
+### European Netherlands: LAT -> ETRS89
+
+*Source*: [NSGI](https://www.nsgi.nl/coordinatenstelsels-en-transformaties/coordinatentransformaties/nllat-nap-etrs89)  
+*Format*: GeoTIFF converted from GTX  
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)  
+
+* nl_nsgi_rdcorr2018.tif
```

Comment:
This is not the filename I see added in this PR: `nl_nsgi_nllat2018.tif`.
Maybe it would be useful to mention the full name of the target CRS: `EPSG:9287` "LAT NL depth" (to see it is a depth one).

Reply:
Good catch! Line 41 of the [nl_nsgi/nl_nsgi_README.txt](https://github.com/OSGeo/PROJ-data/pull/137/files/25563bb4838404093aa1617ee909234f21e76e78#diff-c3d4786ff5d2587993b29cc5809bcf39719f3ce0a6be5e554a606030fa8d0ce7) should be indeed `nl_nsgi_nllat2018.tif`